GameFighter/Models/Warrior.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace GameFighter
+{
+    public class Warrior
+    {
+        public int Health { get; private set; }
```

Comment:
Why the setter is private? How will you set Health property in derived classes?

Reply:
I was sure, that health is a constant variable :))